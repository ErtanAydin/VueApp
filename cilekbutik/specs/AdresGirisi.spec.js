import { mount, RouterLinkStub } from "@vue/test-utils";
import AdresGirisi from "@/components/AdresGirisi";

describe("AdresGirisi", () => {
  test("mounts properly", () => {
    const wrapper = AdresGirisi;
    expect(wrapper).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = mount(AdresGirisi, { 
      stubs: {
        NuxtLink: RouterLinkStub
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});

describe ('data', () => {
  test('data test', () => {
    const wrapper = mount(AdresGirisi,{
    stubs: {
      NuxtLink: RouterLinkStub
    }});
    expect(wrapper.vm.faturaTipi).toBe('bireysel')
    expect(wrapper.vm.faturaDetayi).toBe(false)
    expect(wrapper.vm.sepettekiUrunleriGoster).toBe(false)
  })
})