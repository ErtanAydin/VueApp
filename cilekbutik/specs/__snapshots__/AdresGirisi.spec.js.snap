// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdresGirisi renders properly 1`] = `
<div class="adres-girisi-butun">
  <div class="adres-girisi-content">
    <div class="adres-girisi">
      <div class="adres-header">
        <div class="adres-header-icerik"><i class="fa fa-map-marker adres-icon" style="font-size: 20px;"></i>
          Adres Girişi
        </div>
      </div>
      <div class="adres-body">
        <div class="satir">
          <div class="sutun"><label for="siparisInputName" class="adres-text">
              Ad
              <span class="zorunlu-text">*</span></label> <input type="text" class="input-form"></div>
          <div class="sutun"><label for="siparisInputName" class="adres-text">
              Soyad
              <span class="zorunlu-text">*</span></label> <input type="text" class="input-form"></div>
        </div>
        <div class="satir">
          <div class="sutun"><label for="siparisInputName" class="adres-text">
              E-posta Adresi
              <span class="zorunlu-text">*</span></label> <input type="text" class="input-form"></div>
          <div class="sutun"><label for="siparisInputName" class="adres-text">
              Cep Telefonu
              <span class="zorunlu-text">*</span></label>
            <div><img src="https://cdn.countryflags.com/thumbs/turkey/flag-400.png" width="30" height="20" class="turkbayragi"> <input type="text" class="input-form-cep-tel"></div>
          </div>
        </div>
        <div class="satir">
          <div class="sutun"><label for="siparisInputName" class="adres-text">
              İl
              <span class="zorunlu-text">*</span></label> <select name="sehir" id="siparisInputSehir" class="input-form">
              <option selected="selected" value="">Seçin</option>
              <option value="1">Adana</option>
              <option value="2">Adıyaman</option>
              <option value="3">Afyon</option>
              <option value="4">Ağrı</option>
              <option value="68">Aksaray</option>
              <option value="5">Amasya</option>
              <option value="6">Ankara</option>
              <option value="7">Antalya</option>
              <option value="75">Ardahan</option>
              <option value="8">Artvin</option>
              <option value="9">Aydın</option>
              <option value="10">Balıkesir</option>
              <option value="74">Bartın</option>
              <option value="72">Batman</option>
              <option value="69">Bayburt</option>
              <option value="11">Bilecik</option>
              <option value="12">Bingöl</option>
              <option value="13">Bitlis</option>
              <option value="14">Bolu</option>
              <option value="15">Burdur</option>
              <option value="16">Bursa</option>
              <option value="17">Çanakkale</option>
              <option value="18">Çankırı</option>
              <option value="19">Çorum</option>
              <option value="20">Denizli</option>
              <option value="21">Diyarbakır</option>
              <option value="81">Düzce</option>
              <option value="22">Edirne</option>
              <option value="23">Elazığ</option>
              <option value="24">Erzincan</option>
              <option value="25">Erzurum</option>
              <option value="26">Eskişehir</option>
              <option value="27">Gaziantep</option>
              <option value="28">Giresun</option>
              <option value="29">Gümüşhane</option>
              <option value="30">Hakkari</option>
              <option value="31">Hatay</option>
              <option value="76">Iğdır</option>
              <option value="32">Isparta</option>
              <option value="34">İstanbul</option>
              <option value="35">İzmir</option>
              <option value="46">Kahramanmaraş</option>
              <option value="78">Karabük</option>
              <option value="70">Karaman</option>
              <option value="36">Kars</option>
              <option value="37">Kastamonu</option>
              <option value="38">Kayseri</option>
              <option value="71">Kırıkkale</option>
              <option value="39">Kırklareli</option>
              <option value="40">Kırşehir</option>
              <option value="79">Kilis</option>
              <option value="41">Kocaeli</option>
              <option value="42">Konya</option>
              <option value="43">Kütahya</option>
              <option value="44">Malatya</option>
              <option value="45">Manisa</option>
              <option value="47">Mardin</option>
              <option value="33">Mersin</option>
              <option value="48">Muğla</option>
              <option value="49">Muş</option>
              <option value="50">Nevşehir</option>
              <option value="51">Niğde</option>
              <option value="52">Ordu</option>
              <option value="80">Osmaniye</option>
              <option value="53">Rize</option>
              <option value="54">Sakarya</option>
              <option value="55">Samsun</option>
              <option value="56">Siirt</option>
              <option value="57">Sinop</option>
              <option value="58">Sivas</option>
              <option value="63">Şanlıurfa</option>
              <option value="73">Şırnak</option>
              <option value="59">Tekirdağ</option>
              <option value="60">Tokat</option>
              <option value="61">Trabzon</option>
              <option value="62">Tunceli</option>
              <option value="64">Uşak</option>
              <option value="65">Van</option>
              <option value="77">Yalova</option>
              <option value="66">Yozgat</option>
              <option value="67">Zonguldak</option>
            </select></div>
          <div class="sutun"><label for="siparisInputName" class="adres-text">
              İlçe
              <span class="zorunlu-text">*</span></label> <select name="ilce" id="siparisInputIlce" class="input-form">
              <option value="">Seçin</option>
            </select></div>
        </div>
        <div class="satir">
          <div class="sutun-adres"><label for="siparisInputName" class="adres-text">
              Adres
              <span class="zorunlu-text">*</span></label> <textarea name="Adres" id="siparisInputAdres" class="input-form"></textarea></div>
        </div>
        <div class="satir">
          <div class="fatura-satiri"><input type="checkbox" name="fatura" id="siparisInputFatura"> <label for="siparisInputFatura" class="fatura">Fatura bilgilerimi farklı girmek istiyorum.</label></div>
        </div>
        <!---->
      </div>
    </div>
    <div class="siparis-odeme-info">
      <div class="siparis-ozeti">
        <div class="siparis-ozeti-header">
          <div class="siparis-ozeti-text">
            Sipariş Özeti
          </div>
        </div>
        <div class="siparis-detay-bolumu"><span class="urunleriGosterText"><div class="siparis-sepetteki-urunler-text"><i class="fa fa-caret-down"></i>
                            Sepetteki Ürünler
                        </div></span>
          <!---->
          <div class="sepet-satir">
            <div class="sepet-sutun-sol">
              Ürünler Toplamı
            </div>
            <div class="sepet-sutun-sag"><label id="sagaYapistir">60,00 TL</label></div>
          </div>
          <div class="sepet-satir">
            <div class="sepet-sutun-sol">
              Kargo
            </div>
            <div class="sepet-sutun-sag"><label id="sagaYapistir">12,00 TL</label></div>
          </div>
          <div class="sepet-satir-final">
            <div class="sepet-sutun-sol">
              Genel Toplam <br> (KDV Dahil)
            </div>
            <div class="sepet-sutun-sag" style="vertical-align: center;"><label id="sagaYapistir" style="font-size: 20px; font-weight: bold; color: rgb(49, 171, 69);">72,00 TL</label></div>
          </div>
        </div>
      </div> <button class="button-odeme-islemleri"> ÖDEME İŞLEMLERİ <i class="fa fa-chevron-right"></i></button>
    </div>
  </div>
</div>
`;
